src/quo2/components/community/banner/view.cljs

```diff
@@ -29,16 +29,11 @@
      description]]])
 
 (defn view
-  [{:keys [title description on-press accessibility-label banner]}]
+  [{:keys [title description on-press accessibility-label banner style]}]
   [rn/touchable-without-feedback
    {:on-press            on-press
     :accessibility-label accessibility-label}
-   [rn/view
-    (merge (style/community-card 16)
-           {:background-color (colors/theme-colors
-                               colors/white
-                               colors/neutral-90)}
-           style/banner-card)
+   [rn/view {:style (merge (style/community-card) style)}
```

Comment:
Refactor: :
- that 16 is a parameter, but this function is not used in any other place, so we don't need that it receives a parameter.
-`:background-color` was included with the same values in as the result of calling `style/community-card`
- `style/banner-card` is not used in any other place

So, these 3  maps can be just returned  from the function `style/community-card`.